sotodlib/site_pipeline/make_relcal_model.py

```diff
@@ -0,0 +1,372 @@
+'''This module receives the fitted detector optical response 
+from the analyze-bright-ptsrc after the detmap has been applied, 
+and determines a 'flat field' for the detectors. The script can 
+take one single scan or multiple scans. The final output is a 
+set of per-detector calibration factors of order 1 associated 
+with det_id, to be multiplied to the detector response. 
+'''
+import sys
+import numpy as np
+import os
+import yaml
+import h5py
+from argparse import ArgumentParser
+import matplotlib.pyplot as plt
+from sotodlib.core import Context, metadata
+import sotodlib.io.metadata as io_meta
+from sotodlib.site_pipeline import util
+
+def get_parser():
+
+    parser = ArgumentParser()
+
+    parser.add_argument('config_file', help=
+        'Configuration file.')
+ 
+    parser.add_argument('--obs_id', nargs='*', help=
+         'Observations to summarize over.')
+
```

Comment:
Here and in other args, replace underscore, i.e. `--obs-id` not `--obs_id`.

Also make consistent: `help=` (no spaces), `default=...` (nospaces)